Formula/cadence-workflow.rb

```diff
@@ -22,7 +22,12 @@ class CadenceWorkflow < Formula
   conflicts_with "cadence", because: "both install an `cadence` executable"
 
   def install
-    system "make", ".fake-codegen"
+    # `go` version is hardcoded in Makefile
+    inreplace "Makefile" do |s|
+      s.change_make_var! "EXPECTED_GO_VERSION", Formula["go"].version.to_s
+    end
```

Comment:
only issue to track the `go` upgrade

Reply:
This is all that I found: https://github.com/uber/cadence/issues/5034. The [related PR](https://github.com/uber/cadence/pull/5035) only changed the Dockerfile though.

Reply:
seems not very relevant.